packages/velo-external-db-core/lib/schema_hooks_utils.js

```diff
@@ -0,0 +1,68 @@
+
+const hooksForAction = (action) => {
+    const hooksForAction = {
```

Comment:
This shouldn't be created each time `hooksForAction` is called, this should be a `const HOOKS_FOR_ACTION` in the top of the file.